Automattic/wp-calypso
# New nudges and banners for Black Friday and Cyber Monday sales

#### Changes proposed in this Pull Request

* Add a new nudge and banner for the upcoming Black Friday and Cyber Monday sales.

#### Testing instructions

On the backend, set a user attribute:
```
update_user_attribute( USER_ID, 'SSE_marketing_email_holdout_nov_bf_wpcom_free_20180101', 'send' );
```

In the `active-discounts.js` file, change the `startsAt` date for the `blackfriday30 ` promo to make it active.
Change the month from `10` to `9`:
`startsAt: new Date( 2018, 9, 21, 0, 0, 0 ),`

In Calypso, go to the My Sites section for a site that's currently on free, Personal, or Premium.
You should see the Black Friday nudge:

<img width="288" alt="screen shot 2018-11-16 at 8 01 02 am" src="https://user-images.githubusercontent.com/690843/48632675-5c5fe000-e976-11e8-95ee-892b5e72b89e.png">

Click the nudge and you should see the banner:

<img width="1077" alt="screen shot 2018-11-16 at 8 01 12 am" src="https://user-images.githubusercontent.com/690843/48632682-6255c100-e976-11e8-9f1d-1fbb50a27187.png">

Revert the change to `startsAt` and make the same change to the `cyber30` promo.

Repeat the steps above and you should see the Cyber Monday nudge and banner.

<img width="293" alt="screen shot 2018-11-16 at 8 02 24 am" src="https://user-images.githubusercontent.com/690843/48632701-6c77bf80-e976-11e8-824a-f764b1d35fb5.png">
<img width="1063" alt="screen shot 2018-11-16 at 8 02 37 am" src="https://user-images.githubusercontent.com/690843/48632705-70a3dd00-e976-11e8-98d9-fbcac68cd6f9.png">
